### PR TITLE
chore: upstream code change

### PR DESCRIPTION
**Pull request by Taowyoo**
Upstream the internal latest code to GitHub.

We should deprecate the internal one, and keep maintain this one like we did for https://github.com/fortanix/sdkms-performance-testbench

### CODE REVIEW

File: LICENSE
Comment (mzohreva) [inline LICENSE:1]: Why is the license changed?

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-dsm-perf-tool: *.go cmd/*.go go.*
+sdkms-perf-tool: *.go cmd/*.go go.*
```
Comment (mzohreva) [inline Makefile:1]: The name changes are incorrect. This package is named `dsm-perf-tool`

File: README.md [hunk 2]
```diff
@@ -11,67 +12,5 @@ $ make install
...
-
-(d) I understand and agree that this project and the contribution
-are public and that a record of the contribution (including all
-personal information I submit with it, including my sign-off) is
-maintained indefinitely and may be redistributed consistent with
-this project or the open source license(s) involved.
-
-# License
-
-This project is primarily distributed under the terms of the Mozilla Public License (MPL) 2.0, see [LICENSE](./LICENSE) for details.
```
Comment (mzohreva) [inline README.md:77]: Please revert this change

File: cmd/common.go [hunk 3]
```diff
@@ -1,9 +1,3 @@
-/* Copyright (c) Fortanix, Inc.
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
```
Comment (mzohreva) [inline cmd/common.go:6]: You should keep these license notices and make sure you add them to new files too

File: cmd/testSetup.go [hunk 4]
```diff
@@ -18,8 +12,8 @@ import (
 )
 
 const (
-	DEFAULT_USER          = ""
-	DEFAULT_USER_PASSWORD = ""
+	DEFAULT_USER          = "testuser@fortanix.com"
+	DEFAULT_USER_PASSWORD = "password123"
```
Comment (mzohreva) [inline cmd/testSetup.go:16]: Please don't leave any credentials in the code

File: main.go [hunk 5]
```diff
@@ -1,12 +1,6 @@
-/* Copyright (c) Fortanix, Inc.
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 package main
 
-import "github.com/fortanix/dsm-perf-tool/cmd"
+import "bitbucket.org/fortanix/roche-qa/sdkms-perf-tool/cmd"
```
Comment (mzohreva) [inline main.go:3]: This should point to github

**Comment by mzohreva** [comment]
I don't see any useful change in the PR, I think you can decline this as it does not add any functionality

File: README.md (hunk 2 above)
Comment (Taowyoo) [reply README.md:77]: Sure!

**Comment by Taowyoo** [comment]
You are right, sry for did not check the code change. Going to decline it
